pandas/tests/indexes/test_interval_new.py

```diff
@@ -0,0 +1,310 @@
+from __future__ import division
+
+import pytest
+import numpy as np
```

Comment:
for now add this

``pytestmark = pytest.mark.xfail(....)``
at the top, will xfail this entire module w/o running the individual tests.

Reply:
Will need more specificity here I think.

Reply:
no literally add that with a nice xfail message.

I want this to skip this entire module for now, rather than showing a skip for each test. This just pollutes the xfail output (which is already getting annoying large).

Reply:
so if you add the xfail up top, then you can remove all of the xfails on individual functions.